src/pages/index.astro

```diff
@@ -378,21 +266,118 @@ nf-core list
         </div>
     </div>
 
-    <div id="community" class="homepage-usedby">
-        <div class="container py-5">
-            <h2>
-                <a class="btn btn-success float-end d-none d-md-inline" href="/community#organisations">
-                    See a complete list &raquo;
-                </a>
-                <a href="/community#organisations">Used by groups all over the world</a>
-            </h2>
-            <p>The nf-core community is spread all over the globe and includes a large number of contributing users.</p>
-            <p>
-                <a class="btn btn-success d-inline d-md-none" href="/community#organisations">
-                    See a complete list &raquo;
-                </a>
-            </p>
-            <ContributorCarousel client:visible />
-        </div>
-    </div>
+    <ContributorCarousel client:visible />
 </BaseLayout>
+
+<style lang="scss">
+    @import 'bootstrap/scss/bootstrap.scss';
+    @import 'bootstrap-print-css/css/bootstrap-print.min';
+    @import '../styles/_variables.scss';
+    .homepage-header .homepage-header-contents {
+        margin: 3rem auto 0;
+        padding: 0;
+        text-align: center;
+    }
+    .homepage-header-fa-background {
+        position: absolute;
+        top: 10px;
+        left: 30px;
+        font-size: 16em;
+        opacity: 0.2;
+    }
+
+    /* Fancy call to action button on homepage
+Based on https://codepen.io/wintr/pen/beBJBb */
+    .homepage-cta-flex {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        margin-top: 2rem;
+    }
+    .homepage-cta {
+        width: 100px;
+        height: 100px;
+        line-height: 20px;
+        padding: 30px 0;
+        text-align: center;
+        text-transform: uppercase;
+        letter-spacing: 1px;
+        position: relative;
+        transition: 0.3s all ease;
+        font-size: 14px;
+        color: $nf-core-color;
+        font-weight: bold;
+        z-index: 0;
+    }
+    .homepage-cta:after {
+        transition: 0.3s all ease;
+        content: '';
+        position: absolute;
+        left: 0;
+        top: 0;
+        bottom: 0;
+        right: 0;
+        border-radius: 3px;
+        border: 2px solid $nf-core-color;
+    }
```

Comment:
all this classes should be nested

Reply:
Yup, this was copy + paste from the old SCSS file. I refactored some of it to be nested but not all as it was late 😅  Done it now though.

I think that I wrote this back when the site was old CSS and not SCSS.